### PR TITLE
Pass on direct query parameter during login

### PR DESCRIPTION
**Pull request by rullzer**
If direct is set we should forward it again.
This to make SAML direct login work again.

Signed-off-by: Roeland Jago Douma <roeland@famdouma.nl>

### CODE REVIEW

**Review by blizzz — APPROVED** [review]
looks good

**Review by ChristophWurst — APPROVED** [review]

**Review by skjnldsv — CHANGES_REQUESTED** [review]
See comment

File: core/src/components/login/LoginForm.vue [hunk 1]
```diff
@@ -125,6 +125,9 @@
 			<input type="hidden"
 				name="requesttoken"
 				:value="OC.requestToken">
+			<input type="hidden"
+				name="direct"
+				:value="direct">
```
[the remark this answers is not in the document]
Comment (schiessle) [reply core/src/components/login/LoginForm.vue]: because the "direct" parameter is what we use in various places at the moment, including SAML and GS

**Comment by schiessle** [comment]
tested it successfully!

File: core/src/components/login/LoginForm.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (skjnldsv) [reply core/src/components/login/LoginForm.vue]: yeah, but it's coming as a prop, so we don't care about the name of the variable :)

**Comment by rullzer** [comment]
It will be backported yes

File: core/src/components/login/LoginForm.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (rullzer) [reply core/src/components/login/LoginForm.vue]: @skjnldsv you are right

**Comment by rullzer** [comment]
Ok so i changed the code a bit.
Not direct is only set (and send) if you set direct=1 in the query. Which mimicks more the original behavior.

**Review by skjnldsv — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**